fineract-e2e-tests-runner/src/test/resources/features/LoanChargeback.feature

```diff
@@ -2514,10 +2506,6 @@ Feature: LoanChargeback
       | Type      | Account code | Account name              | Debit | Credit |
       | ASSET     | 112601       | Loans Receivable          |       | 250.0  |
       | LIABILITY | 145023       | Suspense/Clearing account | 250.0 |        |
-    Then Loan Transactions tab has a "ACCRUAL" transaction with date "01 April 2024" which has the following Journal entries:
```

Comment:
Why to remove this?

Reply:
no accrual here, as we do not crate accrual without COB